.circleci/config.yml

```diff
@@ -3,15 +3,29 @@ version: 2.1
 jobs:
   all:
     docker:
-      - image: circleci/node:carbon
+      - image: cypress/browsers:node8.15.1-chrome73
     steps:
       - checkout
-      - run: npm install
+      - restore_cache:
+          keys:
+            - v2-deps-{{ .Branch }}-{{ checksum "package-lock.json" }}
+            - v2-deps-{{ .Branch }}-
+            - v2-deps-
+      - run: npm ci
+      - save_cache:
```

Comment:
It is common for `npm` to produce unstable lock files, especially when you run it on different platforms (e.g. Mac vs Ubuntu in CI).

For this reason using checksum of the lock file for a cache key may not produce right results.

See https://github.com/contentful/doc-app/blob/master/.circleci/config.yml#L26-L38

Reply:
Would you recommend to remove cache at all?

Reply:
See the link - I'd revert any changes made locally in CI to `package-lock.json`